src/NLog/Config/LoggingConfigurationParser.cs

```diff
@@ -1295,9 +1278,122 @@ private static string GetName(Target target)
             return string.IsNullOrEmpty(target.Name) ? target.GetType().Name : target.Name;
         }
 
+        private class ValidatedConfigurationElement : ILoggingConfigurationElement
```

Comment:
I which level this element is "Validated"? I don't fully understand the class name

Reply:
It currently performs validation of attribute-names. But more validation can be added later.